src/types/docker.ts

```diff
@@ -8,6 +8,7 @@ export class DockerExec {
    }
 
    public async pull(image: string, args: string[], silent?: boolean) {
+      image = image.trim()
```

Comment:
this won't actually do anything. trim just removes whitespace which wouldn't affect the command